Casks/o/openhv.rb

```diff
@@ -0,0 +1,28 @@
+cask "openhv" do
+  version "20230917"
+  sha256 "c7f0c0260f690d4039cf7146faad01bf1731b8b77a28eeb0cb9573200c5bc95f"
+
+  url "https://github.com/OpenHV/OpenHV/releases/download/#{version}/OpenHV-#{version}.dmg",
+      verified: "github.com/OpenHV/OpenHV/"
+  name "OpenHV"
+  desc "Pixelart Science-Fiction Real-Time-Strategy game"
+  homepage "https://www.openhv.net/"
+
+  livecheck do
+    url :url
+    regex(/(\d{8})/i)
```

Comment:
```suggestion
    regex(/v?(\d+(?:\.\d+)*)/i)
```

The suggested regex follows our standard implementation.